modules/enabling-seamless-image-access-and-efficient-use-of-multiple-registry-sources.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-seamless-image-access-and-efficient-use-of-multiple-registry-sources_{context}"]
+= Enabling seamless image access and efficient use of multiple registry sources
+
+By setting up image mirroring for registries, you can ensure faster image pulls that enable seamless image access and efficient use of multiple registry sources in your environment.
+
+As a use case, consider a situation where you want to apply the `ImageContentSourcePolicy` custom resource (CR) to an {ocp} cluster that sets up `quay.io` as an image mirror for the `example.com` registry.
+
+.Procedure
+
+. Create a file named `icsp.yaml` by using the following content:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: example
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - quay.io
+    source: example.com
+----
+
+. Apply the YAML file to the {ocp} cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f icsp.yaml
+----
```

Comment:
Setting up mirroring is an OCP feature (not an ACS feature) - instead of adding an example in the ACS docs should we instead refer to (or link to) the OCP docs that show how to setup? - [such as this for OCP 4.16](https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/updating_clusters/performing-a-cluster-update#images-configuration-registry-mirror-configuring_updating-restricted-network-cluster)